### PR TITLE
For nemo/tab switching bufixes jb29532

### PR DESCRIPTION
**Pull request by tworaz**
Bunch of fixes needed to take care of JB#29532

### CODE REVIEW

**Comment by tworaz** [comment]
The embedlite patch was also submitted to upstream embedlite_38 branch. https://github.com/tmeshkova/gecko-dev/pull/67

File: rpm/0016-Enable-external-window-usage.patch [hunk 1]
```diff
@@ -94,11 +113,64 @@ index c12d381..781dcfb 100644
    if (!gl() || gl()->IsDestroyed())
      return false;
  
++  if (gfxPrefs::UseExternalWindow()) {
```
Comment (rainemak) [inline rpm/0016-Enable-external-window-usage.patch:116]: This must be true over here as there is a return already earlier if gfxPrefs::UseExternalWindow was false.

Comment (tworaz) [inline rpm/0016-Enable-external-window-usage.patch:116]: Fixed in next patchset version. Thanks!

**Comment by rainemak** [comment]
Suspending and resuming works nicely. Both when we have an application covering the browser and/or when a tab is backgrounded and a simple page opened to the active tab e.i. about:blank.

Looks good with small changes but another opinion would be good.

@sletta do you have any comments to the gl side?

**Pull request merged**